pkgs/applications/science/biology/spip/default.nix

```diff
@@ -0,0 +1,54 @@
+{ pkgs, stdenv, fetchurl, fetchFromGitHub, lib, rPackages, rWrapper, makeWrapper}:
+
+let
+  my-r-packages = rWrapper.override{packages = with rPackages; [
+    foreach
+    doParallel
+    randomForest
+  ];};
+  transcriptome-url = "https://kumisystems.dl.sourceforge.net/project/splicing-prediction-pipeline/transcriptome/";
+
+  transcriptome19 = fetchurl {
+    url = transcriptome-url + "/transcriptome_hg19.RData";
+    sha256 = "sha256-E8UmHoNoySSIde+TRE6bxVP0PrccpKDvIHBGCvWnYOw=";
+  };
+
+  transcriptome38 = fetchurl {
+    url = transcriptome-url + "/transcriptome_hg38.RData";
+    sha256 = "sha256-mQMMZVN1byXMYjFoRdezmoZtnhUur2CHOP/j/pmw8as=";
+  };
```

Comment:
Note to other reviewers: downloading these files takes way longer than expected for 800 MB.